docs/ManualDeploymentGuideScalarDLonAWS.md

```diff
@@ -0,0 +1,287 @@
+# Deploy Scalar DL on AWS
+
+Scalar DL is a database-agnostic distributed ledger middleware containerized with Docker. 
+It can be deployed on various platforms and it is recommended to be deployed on managed services for production to achieve high availability and scalability, and maintainability. 
+This guide shows you how to manually deploy Scalar DL on a managed database service and a managed Kubernetes service in Amazon Web Services (AWS) as a starting point of deploying Scalar DL for production.
+
+## What we create
+
+![image](images/network_diagram_eks.png)
+
+In this guide, we will create the following components.
+
+* A VPC with NAT gateway
+* An EKS cluster with two Kubernetes node groups
+* A managed database service (you can choose one of them)
+    * DynamoDB
+    * RDS (MySQL/PostgreSQL)
+* 1 Bastion instance with a public IP
+* Amazon CloudWatch
+
+## Step 1. Configure your network
+
+Configure a secure network with your organizational or application standards. 
+Scalar DL handles highly sensitive data of your application, so you should create a highly secure network for production. This section will help you to configure a secure network for Scalar DL deployments.
+
+> **REQUIREMENTS:**
```

Comment:
Why using quote? Wrong usage.
Write them as they are.
All capital looks weird for this,.

Reply:
> Why using quote? Wrong usage.

Sorry, the use of block quotes for requirements and recommendations was wrong.
I used it to point out important points but it was wrong in the MD files.

> Write them as they are.
All capital looks weird for this,.

Yes, now I changed that accordingly.
And I moved Note to  block quotes because `Important information that is not part of the main document flow should be added as a Note in bold with all content contained within a block quote`